integr8ly/application-monitoring-operator
# Intly 2810

The `origin-prometheus-operator:4.2` image is not compatible with the version of the prometheus operator deployed in openshift monitoring. The two operators interfere and this causes openshift-monitoring to break. Using the coreos community image resolves this problem.

This fix is to unblock other work, but we should investigate why the two operators interfere in the first place.

ping @camilamacedo86

Verification steps:

1. Install middleware monitoring but change the version of the operator in the manifest to 0.0.22
2. Installation should succeed
3. Make sure that all pods in middleware-monitoring come up
4. Make sure that all pods in openshift-monitoring are working
